src/components/Anchor/styles/ExternalIconAnchor.js

```diff
@@ -8,7 +8,6 @@ import { base, positive, negative, dark, inverted } from './sharedStyles';
 
 const iconStyles = css`
   text-decoration: none;
-  display: inline-block;
```

Comment:
This seems like it has the possibility of breaking something

Reply:
I did a run through and realized I needed `auto` on the horizontal spacing by default. After that, everything seemed to work properly. I added visual comparisons as well to see what's going on and demonstrated the anchor's use within other text.